smart-contracts/test/UniswapOracleV3/unlockOracleGNP.mainnet.js

```diff
@@ -23,10 +23,8 @@ describe('Unlock GNP conversion', () => {
   let WETH, USDC
 
   before(async function () {
-    if (!process.env.RUN_FORK) {
-      // all suite will be skipped
-      this.skip()
-    }
```

Comment:
this should be kept as we will only run these tests on a fork of mainnet. We don't want to run all tests on forks, only the one with a `*.mainnet.js` extension